src/ResourceManagement/AppService/Generated/Models/IpSecurityRestriction.cs

```diff
@@ -29,13 +21,30 @@ public IpSecurityRestriction()
         /// Initializes a new instance of the IpSecurityRestriction class.
         /// </summary>
         /// <param name="ipAddress">IP address the security restriction is
-        /// valid for.</param>
+        /// valid for.
```

Comment:
@dpetillo 
you can undo these changes, The model got updated with these properties added to it :
https://github.com/Azure/azure-libraries-for-net/blob/303240ff67225bf622b29cdaa0a6268e37bb5c63/src/ResourceManagement/AppService/Generated/Models/IpSecurityRestriction.cs#L51